CollapseLauncher/Classes/InstallManagement/BaseClass/InstallManagerBase.cs

```diff
@@ -481,17 +483,168 @@ public async Task MoveGameLocation()
         public async ValueTask<bool> UninstallGame()
         {
             string GameFolder = ConverterTool.NormalizePath(_gamePath);
-
             switch (await Dialog_UninstallGame(_parentUI, GameFolder, _gameVersionManager.GamePreset.ZoneFullname))
```

Comment:
The implementation is straight-forward but I think it can be improved (but this is optional):

Can we move the switch assignment to override method for each ``*Install`` classes?
But we can keep the deletion process below the ``switch`` statement.